README.md

```diff
@@ -20,19 +20,35 @@ https://puppet.com/docs/pe/latest/getting_support_for_pe.html#pe-support-script
 
 ### Install the `healthcheck_lite` module
 
-To install the `healthcheck_lite` module, execute the following command on the Primary Master.
+#### Using Bolt
+
+Bolt is the preffered installation and execution method and can be used either on the master or another machine with connectivity to the master. Firstly, follow the instruction for [installing Bolt](https://puppet.com/docs/bolt/latest/bolt_installing.html). PE customers are *strongly* encouraged to follow the note about directly installing the package if installing on a PE node.
+
+The following block of code will create a Boltdir under the logged in user's home directory and set up the module:
 
 ```bash
-puppet module install puppetlabs-healthcheck_lite --modulepath=/opt/puppetlabs/puppet/modules
+mkdir -p ~/Boltdir
+cd !$
+
+cat >Puppetfile <<EOF
+mod 'puppetlabs-stdlib'
+mod 'puppetlabs-healthcheck_lite'
+EOF
+
+bolt puppetfile install
 ```
 
-Doing so will install this module into the base module path, making its tasks available without interfering with other modules.
+#### Manually
```

Comment:
Installing the `healthcheck_lite` module via `bolt puppetfile install` or `puppet module install` are both command line actions, so the "Using Bolt" vs "Manual" distinction is misleading. "Manual" should be "Using Puppet Task".

Reply:
On the Forge we say `To manually install this module` in regards to `puppet module install`, so I wouldn't say it's misleading.  How about we change it from `Manually` to `Via Puppet Task` in the `Usage` section?  To me that makes more sense.

Reply:
Yes, that removes the interpretation.